gbpservice/nfp/service_vendor_agents/vyos/oc_config_server/visibility_api.py

```diff
@@ -0,0 +1,55 @@
+# One Convergence, Inc. CONFIDENTIAL
```

Comment:
Lets not call this as visibility_api. Make it generic. We can discuss on the naming.